app/ports/templates/ports/port-detail/installation_stats.html

```diff
@@ -256,10 +283,40 @@ <h4 class="modal-title">How are monthly installations calculated?</h4>
 
         var options = {
             'title': 'XCode Versions',
-            isStacked: true
+            isStacked: true,
+            chartArea: {left: 50, width: "65%"}
+
             };
 
         var chart = new google.visualization.ColumnChart(document.getElementById('xcode-chart'));
         chart.draw(data, options);
         }
     </script>
+
+    <!-- CLT VERSIONS -->
+    <script type="text/javascript">
+        google.charts.load('current', {'packages': ['corechart']});
+
+        google.charts.setOnLoadCallback(drawChart);
+
+        function drawChart() {
+            {% regroup port_installations_by_os_and_clt_version|dictsort:"submission__clt_version" by submission__clt_version as clt_parent %}
+            {% regroup port_installations_by_os_and_clt_version by submission__os_version as os_clt_parent %}
+
+            var data = google.visualization.arrayToDataTable([
+                ['OS Version', {% for version in clt_parent %}'{% if version.grouper == '' %}unknown{% else %}{{ version.grouper }}{% endif %}', {% endfor %} { role: 'annotation' } ],
+                {% for os in os_clt_parent %}
+                    ['OS: {{ os.grouper }}', {% for version in clt_parent %}{% for i in os.list %}{% if i.submission__clt_version == version.grouper %}{{ i.num }}{% endif %}{% endfor %}, {% endfor %} ''],
+                {% endfor %}
+        ]);
+
+        var options = {
+            'title': 'CLT Versions',
```

Comment:
Let's spell that name out.

I would use singular for version all over the place, but that's unrelated

Reply:
Okay, thanks.